src/app/models/book-toc.ts

```diff
@@ -3,13 +3,10 @@ import {cnxFetch} from './table-of-contents-html';
 import memoize from 'lodash/memoize';
 
 export function bookToc(slug: string) {
-    if (!slug) {
-        return Promise.reject(new Error('No slug to fetch'));
-    }
     return cmsFetch(slug)
         .then((bi) => {
-            const isRex = Boolean(bi.webview_rex_link);
-            const webviewLink = isRex ? bi.webview_rex_link : bi.webview_link;
+            const isRex = true;
```

Comment:
I see that this controls which paths to use but I'm not following why it can always be true

Reply:
This is leftover from the transition to Rex. The non-Rex option was CNX, which no longer exists.
There's no longer a `webview_link` field in the CMS. I need to go through the codebase and remove `isRex` generally.